services/src/atdd/src/test/resources/features/mvp.1/FlowReinstall.feature

```diff
@@ -0,0 +1,64 @@
+@FREINSTALL
+Feature: Flow re-reinstalling after switch comes back up.
+
+  @MVP1.1
+  Scenario: Re-installing Ingress and Egress flows.
+
+  This scenario creates simple network and ingress/egress flows. Next this scenario turns off and on back, then checks whether
+  flows were re-installed again.
+
+    Given a clean flow topology
+    And a clean controller
+    And a clean flow topology
```

Comment:
Why do we have "a clean flow topology" twice?
If we ahve a clean controller, don't we have a clean flow topology? (maybe not, but seems like we should)